CHANGELOG.md

```diff
@@ -7,27 +7,106 @@ This release extends the planned support of the modules to OneView REST API vers
 - oneview_id_pools_ipv4_subnets_facts
 
 ## v6.0.0 (unreleased)
```

Comment:
The same statement is present above. merge has to be done.

Reply:
removed it